src/content/docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring.mdx

```diff
@@ -15,25 +15,23 @@ redirects:
 
 Browser monitoring includes [strict security measures](/docs/browser/new-relic-browser/performance-quality/security-new-relic-browser) to provide a robust, standalone product with browser monitoring features. Before you [install the browser agent](/docs/browser/new-relic-browser/getting-started/adding-apps-new-relic-browser), make sure your system meets these requirements.
 
-Want to try out our browser monitoring agent? [Create a New Relic account](https://newrelic.com/signup)... It's for free, forever! No credit card required.
-
 ## Basic requirements [#requirements]
 
-To report data to browser, your application must at a minimum meet two basic requirements:
+To report data to browser, your application must meet at a minimum two basic requirements:
 
 1. The traffic to the application must be from clients able to load and execute browser JavaScript code and set cookies.
```

Comment:
Forgot to take this part about cookies out since we no longer use cookies.
```suggestion
1. The traffic to the application must be from clients able to load and execute browser JavaScript code.
```